splunk-otel-android/src/main/java/com/splunk/rum/Config.java

```diff
@@ -36,27 +37,33 @@
     private final boolean anrDetectionEnabled;
 
     private Config(Builder builder) {
-        this.beaconUrl = builder.beaconUrl;
-        this.rumAuthToken = builder.rumAuthToken;
+        this.beaconEndpoint = builder.beaconEndpoint;
+        this.rumAuthToken = builder.rumAccessToken;
         this.debugEnabled = builder.debugEnabled;
         this.applicationName = builder.applicationName;
         this.crashReportingEnabled = builder.crashReportingEnabled;
-        this.globalAttributes = builder.globalAttributes;
+        Attributes globalAttributes = builder.globalAttributes;
+        if (builder.deploymentEnvironment != null) {
+            globalAttributes = globalAttributes.toBuilder()
+                    .put(ResourceAttributes.DEPLOYMENT_ENVIRONMENT, builder.deploymentEnvironment)
+                    .build();
+        }
+        this.globalAttributes = globalAttributes;
         this.networkMonitorEnabled = builder.networkMonitorEnabled;
         this.anrDetectionEnabled = builder.anrDetectionEnabled;
     }
 
     /**
      * The configured "beacon" URL for the RUM library.
      */
-    public String getBeaconUrl() {
-        return beaconUrl;
+    public String getBeaconEndpoint() {
+        return beaconEndpoint;
     }
 
     /**
      * The configured RUM auth token for the library.
      */
-    public String getRumAuthToken() {
+    public String getRumAuth() {
```

Comment:
Should it be `getRumAuthToken()`?

Reply:
I think it should actually be `getRumAccessToken()`. ;)

Reply:
fixed.